Packages/manifest.json

```diff
@@ -0,0 +1,51 @@
+{
+  "scopedRegistries": [
+    {
+      "name": "npmjs",
+      "url": "https://registry.npmjs.org/",
+      "scopes": [
+        "io.extendreality"
+      ]
+    }
+  ],
+  "dependencies": {
+    "com.unity.ads": "2.0.8",
+    "com.unity.analytics": "3.2.2",
+    "com.unity.collab-proxy": "1.2.15",
+    "com.unity.package-manager-ui": "2.0.7",
+    "com.unity.purchasing": "2.0.3",
+    "com.unity.textmeshpro": "1.3.0",
+    "com.unity.xr.oculus.standalone": "1.38.3",
+    "io.extendreality.vrtk.prefabs": "1.0.1",
```

Comment:
Move this up in `dependencies` so it's the first entry instead.

Reply:
Unity auto changes this, I did put it first and unity swapped it

Reply:
i changed it anyway, but nothing to say unity wont swap it again for other people.